rest-api/etl/run_cloud_etl.sh

```diff
@@ -25,9 +26,50 @@ CREDS_ACCOUNT=${ACCOUNT}
 
 source tools/auth_setup.sh
 
+DATE_WITH_TIME=`date "+%Y%m%d-%H%M%S"`
+INSTANCE_NAME=rdr-etl${DATE_WITH_TIME}
+
+INSTANCE_ARGS=""
+if [ -z "${NOCLONE}" ]
+then
+  set -e
+  echo "Creating clone instance ${INSTANCE_NAME}..."
+  gcloud sql instances clone rdrmaindb ${INSTANCE_NAME}
```

Comment:
So far, yes. I have tested with staging, in which this runs for several minutes and exits when complete. I have not yet tested with a larger database; it's possible it will exit before it's done on a larger database. (I have seen this with gcloud sql import statements, for example.) I can't find any documentation about it either way, so I think we'll have to try it and see.